### PR TITLE
Fix client infinite reloading - Load main.js without cache

### PR DESCRIPTION
**Pull request by beshur**
- Programmatically load main.js with a query param key to avoid browser caching
- Move libs to /libs folder
- Remove unused jQuery version

This should fix page reload cycle.
The reason for it to happen is that somehow on the server side, the `main.js` gets prepended on the fly with the `sid = ...` line which probably stands for session id. But since `main.js` is a static file, it gets cached by browser therefore it loads with invalid session id.
If there is a way to fix this on server-side, that would be better than this ugly hack. In the meantime it should work.

### CODE REVIEW

File: src/fah/webclient/index.html [hunk 1]
```diff
@@ -436,7 +436,7 @@ <h2>Welcome to Folding@home. Please read this.</h2>
 	  <div class="col-50">
 		<ul class="radio-selector">
 		  <li>
-		    <input id="account_type1" name="account_type" type="radio"
```
Comment (beshur) [inline src/fah/webclient/index.html]: dangling space

**Comment by ngood97** [comment]
This change should fix a glaring issue with the client, it would be great to see it merged in!

**Comment by lallenlowe** [comment]
LGTM

**Comment by jcoffland** [comment]
Thanks.  This is the right fix.

**Comment by Gabanders** [comment]
How do I get this new web client version that fixes the page reload cycle? I see that the version number has been raised to 7.6.0 but the version available to download from the FAH web is still 7.5.1

**Comment by beshur** [comment]
@Gabanders, need to wait until there is web-client deployment coming out.

**Comment by cprivite** [comment]
Is this not good enough reason for a new deployment on its own?

**Comment by pyrite357** [comment]
> Is this not good enough reason for a new deployment on its own?

Totally freakin agree. No point waiting. my guess is, something else isn't ready

**Pull request merged**